server/config/routes.js

```diff
@@ -69,6 +69,8 @@ var subsidies            = require('../controllers/categorised/subsidies');
 var units                = require('../controllers/units');
 var transfers           = require('../controllers/finance/transfers');
 
+var service    = require('../controllers/admin/service');
```

Comment:
This should be plural to fit with the rest of our controllers (`services`).  Both the name of the controller and the variable name.